### DISCUSSION TITLE
v1.10 ClickHouse primary key change - How to implement?

### DISCUSSION THREAD

**Discussion by MattKobayashi** [akvorado/akvorado#1164] [Q&A]
Hi all,

The release notes for Akvorado v1.10 mention a change to the primary key used in the ClickHouse main table, and provides a link to [this article](https://kb.altinity.com/altinity-kb-schema-design/change-order-by/) for information on how to apply the change to existing installations, but the article is very generic and I'm not able to find any information on the specific method for how to do this in Akvorado. Does anyone have a guide on how to apply this change to existing installs?

**Comment by vincentbernat** [comment]
The schema change can be found in this commit: https://github.com/akvorado/akvorado/commit/74c150ba52297c8a97a8af8d99b21e2e242cf1fd. You need to change `TimeReceived` to `toStartOfFiveMinutes(TimeReceived)`.
